internal/seclang/rule_parser.go

```diff
@@ -172,10 +172,8 @@ func (p *RuleParser) ParseOperator(operator string) error {
 	// We clone strings to ensure a slice into larger rule definition isn't kept in
 	// memory just to store operator information.
 	opRaw, opdataRaw, _ := strings.Cut(operator, " ")
-	opRaw = strings.Clone(opRaw)
```

Comment:
Since we have to store the raw rule anyways, now that the slices point to it the cloning isn't needed